azure-iot-device/azure/iot/device/common/config.py

```diff
@@ -0,0 +1,17 @@
+import logging
```

Comment:
This file should arguably be moved to the pipeline subpackage, but I suppose it's not crucial to do right this minute.